sphinx/texinputs/latexmkjarc

```diff
@@ -0,0 +1,7 @@
+$latex = 'platex --halt-on-error --interaction=nonstopmode -kanji=utf8 %O %S';
+$dvipdf = 'dvipdfmx %O -o %D %S';
+$makeindex = 'rm -f %D; mendex -U -f -d %B.dic -s python.ist %S || echo "mendex exited with error code $? (ignoring)" && : >> %D';
```

Comment:
I did the `rm` based on a comment by Latexmk author:

> Actually, there's one problem: if there's a previously existing .ind file, it will not be deleted.  In that case you end up with an .ind file with wrong information in it instead of a zero length file..

Thus it seems we are considering here a situation where
- we have some ind file handing around
- the tex file in the end contains no `\index` command hence would generate non ind file by itself

Thus perhaps for simplicity sake it is better not to worry. It should be rare when suddenly all `\index` have disappeared. Hence let's just drop this (if we can find a working solution to other things).